mobile/status.go

```diff
@@ -1394,25 +1686,42 @@ func ToChecksumAddress(address string) string {
 }
 
 func DeserializeAndCompressKey(DesktopKey string) string {
+	return logAndCallString(deserializeAndCompressKey, DesktopKey)
+}
+
+func deserializeAndCompressKey(DesktopKey string) string {
 	deserialisedKey := MultiformatDeserializePublicKey(DesktopKey, "f")
 	sanitisedKey := "0x" + deserialisedKey[5:]
 	return CompressPublicKey(sanitisedKey)
 }
 
+type InitLoggingRequest struct {
+	logutils.LogSettings
+	LogRequestGo   bool
+	LogRequestFile string
+}
+
 // InitLogging The InitLogging function should be called when the application starts.
 // This ensures that we can capture logs before the user login. Subsequent calls will update the logger settings.
 // Before this, we can only capture logs after user login since we will only configure the logging after the login process.
 func InitLogging(logSettingsJSON string) string {
-	var logSettings logutils.LogSettings
+	var logSettings InitLoggingRequest
```

Comment:
This is a breaking change, right? Please don't forget to follow [How to introduce breaking changes](https://www.notion.so/How-to-introduce-breaking-changes-ded9ec2d91464a46a2593c0d8de62fbe?pvs=4).

Reply:
Oh! The new type is "inherited" from the old one, my bad 👍